easybuild/easyconfigs/v/ViennaRNA/ViennaRNA-2.7.0-foss-2023a.eb

```
@@ -0,0 +1,60 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'ConfigureMake'
+
+name = 'ViennaRNA'
+version = '2.7.0'
+
+homepage = 'https://www.tbi.univie.ac.at/RNA/'
+description = """The Vienna RNA Package consists of a C code library and several
+stand-alone programs for the prediction and comparison of RNA secondary structures."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://www.tbi.univie.ac.at/RNA/download/sourcecode/%(version_major)s_%(version_minor)s_x/']
+sources = [SOURCE_TAR_GZ]
+patches = ['%(name)s-%(version)s-flexiblas-for-RNAxplorer.patch']
+checksums = [
+    {'ViennaRNA-2.7.0.tar.gz': '9a99fd68ed380894defb4d5e6a8a2871629270028cdf7f16f0a05da6e8c71473'},
+    {'ViennaRNA-2.7.0-flexiblas-for-RNAxplorer.patch':
+     'a5c71e67193d489b9c4f2be6467da5d760f15ad562bb29c5c57f5ae38b3c51b3'},
+]
+
+builddependencies = [
+    ('Autotools', '20220317'),
+    ('flex', '2.6.4'),
+    ('pkg-config', '0.29.2'),
```

can we use `pkgconf` instead? For recent toolchains, we should only use `pkg-config` if we have issues with `pkgconf`
```suggestion
    ('pkgconf', '1.9.5'),
```